_pages/dev/styling-and-page-layout/directionality.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Directionality
+---
+
+{% capture version_note %}
+{{ site.version_note_part1 }} 2.1 {{ site.version_note_part2 }}
+{% endcapture %}
+
+{% include docs/feature_version.html content=version_note %}
+
+The directionality feature provides support for bidirectional text and layout. You can configure Spartacus to use a left-to-right (LTR) orientation, or a right-to-left (RTL) orientation.
+
+Directionality is driven by language. Many languages are read from left to right, but well known examples, such as Arabic and Hebrew, are read from right to left.
+
+In Spartacus, the direction of the UI reflects the active language, so that directionality can work in a multi-directional experience. If your storefront contains both LTR and RTL languages, the active language is used to detect the direction automatically.
+
+## Breaking Changes
+
+The changes made to the DOM and CSS are considered breaking changes, so the directionality feature can only be used if you enable it with the 2.1 feature flag and the 2.1 CSS version.
+
+## Configuring Directionality
+
+You can configure directionality with the properties from the `DirectionConfig` interface. The default configuration contains the following properties:
+
+```typescript
+const defaultDirectionConfig: DirectionConfig = {
+  direction: {
+    detect: true,
+    default: DirectionMode.LTR,
+    rtlLanguages: ["he", "ar"],
+  },
+};
+```
+
+With the default configuration, all languages are mapped to the LTR direction, except for Hebrew (he) and Arabic (ar). Additional RTL languages can be added with configuration.
+
+The default configuration should be suitable for most projects, but if you implement a storefront with an RTL orientation, you might consider changing the default direction to RTL and introducing some explicit LTR languages. The following is an example:
+
+```typescript
+ConfigModule.withConfig({
+    direction: {
+        default: DirectionMode.RTL,
+        ltrLanguages: ['en'],
+    },
+} as DirectionConfig),
+```
+
+For more information on configuration, see [Global Configuration in Spartacus]({{ site.baseurl }}{% link _pages/dev/global-configuration-in-spartacus.md %}).
+
+## Implementing Directionality
```

Comment:
I don't like that title. What about implementation details?

Reply:
👍